src/test/java/net/snowflake/client/jdbc/SnowflakeDriverIT.java

```diff
@@ -2913,6 +2915,13 @@ public void testPutGet() throws Throwable {
         try {
           statement.execute("CREATE OR REPLACE STAGE testPutGet_stage");
 
+          if (!testS3stageRegionalUrl
```

Comment:
Could you edit this test and try setting this param for all types of accounts just to make sure it doesn't break gcp, azure, etc?

Reply:
This flag is not set in other deployments, But enabling it across all cloud providers to make sure it doesnt break others.